### PR TITLE
fix(app): fix robot system version screen and modal

### PR DESCRIPTION
**Pull request by koji**
# Overview
design
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen?type=design&node-id=7808-153582&t=7D5CGTa3tzrSx4zN-0

update robot system version screen and modal.
for `check updates` in robot system version isn't implemented because we don't have that function and also odd only supports update and inline notification tells display the information when there is update.

Also close RAUT-454 and update Update Channel's header
 

# Test Plan

# Changelog
- update robot system version and modal + test cases
- move `RobotSettingsDashboard` and update tests
- move `Navigation` folder and update tests

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/RobotSettingsDashboard/RobotSystemVersionModal.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <Modal header={modalHeader} modalSize="large">
+      <Flex
+        flexDirection={DIRECTION_COLUMN}
+        gridGap={SPACING.spacing32}
+        width="100%"
+      >
+        <Flex
+          flexDirection={DIRECTION_COLUMN}
+          gridGap={SPACING.spacing24}
+          overflow={OVERFLOW_SCROLL}
```
Comment (brenthagen) [inline app/src/organisms/RobotSettingsDashboard/RobotSystemVersionModal.tsx]: scroll axes are showing on dev kit even when no release notes (no overflow) are present. `OVERFLOW_AUTO` should only show scroll axes when overflow exists

File: app/src/organisms/RobotSettingsDashboard/RobotSystemVersionModal.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        >
+          <InlineNotification
+            type="neutral"
+            heading={t('updating_robot_system')}
+            hug
+          />
+          {/* </Flex> */}
```
Comment (brenthagen) [inline app/src/organisms/RobotSettingsDashboard/RobotSystemVersionModal.tsx]: comment

File: app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx [hunk 3]
```diff
@@ -57,84 +57,81 @@ export function RobotSystemVersion({
...
+              <MediumButton
+                flex="1"
+                buttonText={t('view_update')}
+                onClick={() => setShowModal(true)}
+              />
+            ) : (
+              <MediumButton
+                flex="1"
+                buttonText={t('check_for_updates')}
+                onClick={() => console.log('check for updates')}
```
Comment (brenthagen) [inline app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx]: should push to `/robot-settings/update-robot` ?

File: app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx [hunk 4]
```diff
@@ -57,84 +57,81 @@ export function RobotSystemVersion({
...
-              fontSize="1.5rem"
-              lineHeight="2.0625rem"
-              fontWeight={TYPOGRAPHY.fontWeightRegular}
-            >{`${t(
-              'shared:view_latest_release_notes'
+            <StyledText as="p">{`${t(
+              'view_latest_release_notes_at'
             )} ${GITHUB_URL}`}</StyledText>
             <Flex
               backgroundColor={COLORS.light2}
```
Comment (brenthagen) [inline app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx]: `COLORS.light1`

File: app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx [hunk 5]
```diff
@@ -57,84 +57,81 @@ export function RobotSystemVersion({
...
+          </Flex>
           {isUpdateAvailable ? (
             <Flex
               flexDirection={DIRECTION_ROW}
-              padding={`${SPACING.spacing24} ${SPACING.spacing16}`}
+              padding={`${SPACING.spacing12} ${SPACING.spacing16}`}
               gridGap={SPACING.spacing16}
               borderRadius={BORDERS.borderRadiusSize3}
               alignItems={ALIGN_CENTER}
               backgroundColor={COLORS.warningBackgroundMed}
```
Comment (brenthagen) [inline app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx]: `COLORS.yellow3`

**Review by brenthagen — APPROVED** [review]
just some minor comments. once conflicts fixed LGTM !

File: app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx (hunk 3 above)
Comment (brenthagen) [reply app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx]: or maybe we should remove the button if we don't have the capability to do a check

Comment (koji) [reply app/src/organisms/RobotSettingsDashboard/RobotSystemVersion.tsx]: We don't have check for updates function now. 
Asking about this to Mel.

**Pull request merged**